### PR TITLE
[DOCFIX] Add javadoc for JournalShutdownIntegrationTest#awaitClientTermination

### PR DESCRIPTION
**Pull request by Popeye13**

### CODE REVIEW

File: tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java [hunk 1]
```diff
@@ -118,7 +118,10 @@ public void multiMasterJournalStopIntegration() throws Exception {
     // clean up
     cluster.stopUFS();
   }
-
+
```
Comment (apc999) [inline tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java]: unnecessary whitespace in line 121.

**Comment by yupeng9** [comment]
Jenkins, test this please
